### PR TITLE
Add 'exisa' to help message for infoset types.

### PR DESCRIPTION
**Pull request by jadams-tresys**
Missed this during the initial implementation.

DAFFODIL-1959

### CODE REVIEW

File: daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala [hunk 1]
```diff
@@ -313,7 +313,7 @@ class CLIConf(arguments: Array[String]) extends scallop.ScallopConf(arguments) {
 
     val config = opt[File](short = 'c', argName = "file", descr = "XML file containing configuration items")
     val vars = props[String](name = 'D', keyName = "variable", valueName = "value", descr = "Variables to be used when parsing. Can be prefixed with {namespace}.")
-    val infosetType = opt[InfosetType.Type](short = 'I', argName = "infoset_type", descr = "Infoset type to output. Use 'xml', 'scala-xml', 'json', 'jdom', 'w3cdom', 'sax', 'exi', or 'null'. Defaults to 'xml'.", default = Some(InfosetType.XML))
+    val infosetType = opt[InfosetType.Type](short = 'I', argName = "infoset_type", descr = "Infoset type to output. Use 'xml', 'scala-xml', 'json', 'jdom', 'w3cdom', 'sax', 'exi', 'exisa', or 'null'. Defaults to 'xml'.", default = Some(InfosetType.XML))
```
Comment (stevedlawrence) [inline daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala:316]: Thoughts on changing this to something like:

```scala
descr = "Infoset type to output. Type can be: " + InfosetType.values.mkString(", ") + ". Defaults to xml".
```

This way we don't need to remember to update the help everytime a new infoset type is added?

**Review by stevedlawrence — APPROVED** [review]
+1

File: daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala (hunk 1 above)
Comment (jadams-tresys) [reply daffodil-cli/src/main/scala/org/apache/daffodil/Main.scala:316]: Yup, I like that change.  Will do.

**Review by tuxji — APPROVED** [review]
+1

Good suggestion, Steve.

**Pull request merged**